walkeeper/src/lib.rs

```diff
@@ -22,6 +22,9 @@ pub mod defaults {
     pub const DEFAULT_HTTP_LISTEN_ADDR: &str = formatcp!("127.0.0.1:{DEFAULT_HTTP_LISTEN_PORT}");
 }
 
+// PG timeline is always 1, changing it doesn't have useful meaning in Zenith.
+pub const PG_TLI: u32 = 1;
```

Comment:
Please, move this definition to `postgres_ffi/xlog_utils.rs`, so that we could use it in pageserver too.
We have a couple of places with comments about this tli: `// FIXME: always use Postgres timeline 1`

Reply:
done